packages/server/serverless.yml

```diff
@@ -13,8 +21,34 @@ functions:
           path: /trpc/{proxy+}
           method: any
           cors: true
+    environment:
+      PRISMA_DATABASE_URL: ${file(${self:custom.envFile}):PRISMA_DATABASE_URL}
+
+package:
+  # TODO: this is gross....
+  patterns:
+    - ${self:custom.prismaEngine.${self:custom.stage}}
+    - "./../../node_modules/.pnpm/@prisma+client@5.0.0/node_modules/.prisma/client/schema.prisma"
```

Comment:
this is too deep is this necessary?

Reply:
it doesn't even work.  I'm just going to switch to `npm` because `pnpm` is doing this weird.